tools/ovextcap/main.c

```diff
@@ -0,0 +1,334 @@
+/*
+ * Copyright (c) 2019 Tomasz Moń <desowin@gmail.com>
+ *
+ * SPDX-License-Identifier: GPL-2.0-or-later
+ */
+
+#define _POSIX_C_SOURCE 199309L
+#include <getopt.h>
+#include <stdbool.h>
+#include <stdint.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <time.h>
+
+#include <ov.h>
+
+#define EXTCAP_VERSION_STR "0.0.1"
+
+#define PCAP_NANOSEC_MAGIC   (0xa1b23c4d)
+#define LINKTYPE_USBLL              (288)
+
+#define OV_TIMESTAMP_FREQ_HZ   (60000000)
+
+struct handler_data
+{
+    FILE *out;           /**< Output file. Set to NULL if capture stopped from Wireshark (broken pipe). */
```

Comment:
Would it be possible for you to describe the style in .astylerc or similar?

Reply:
Well, It is a good reason for me to get learn some syntax check tools! :)
But what do you think about `clang-format` instead of `astylerc`? My colleagues use `clang-format`.

Reply:
Choose whatever you prefer. As long as I can just run the tool over the source file I don't mind.

Reply:
I've just put `.clang-format` into `master`.